x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/components/agent_activity_flyout.tsx

```diff
@@ -143,7 +143,7 @@ export const AgentActivityFlyout: React.FunctionComponent<{
                   body={
                     <FormattedMessage
                       id="xpack.fleet.agentActivityFlyout.noActivityDescription"
-                      defaultMessage="Activity feed will appear here as agents get enrolled, upgraded, or configured."
+                      defaultMessage="Activity feed will appear here as agents are reassigned, upgraded, or unenrolled."
```

Comment:
tweaked description to fix https://github.com/elastic/kibana/issues/141186

<img width="515" alt="image" src="https://user-images.githubusercontent.com/90178898/191534223-2b3a734a-c0f2-4c18-9e06-931865356a05.png">